python/ingestion/dataset_utils.py

```diff
@@ -37,6 +37,22 @@ def calc_pct_share(record, total_value):
     return pd.concat(with_pct_share).reset_index(drop=True)
 
 
+def percent_avoid_rouding_to_zero(numerator, denominator, default_decimals=1, max_decimals=2):
+    """Calculates percentage to `default_decimals` number of decimal places. If
+       the percentage would round to 0, calculates with more decimal places until
+       either it doesn't round to 0, or until `max_decimals`. `defaultDecimals`
+       and `maxd_decimals` should be >= 0 and `max_decimals` should be >=
```

Comment:
this is supposed to be `max_decimals` not `maxd_decimals` right?

Reply:
Yup thanks